### DISCUSSION TITLE
[Off-Topic] Source of heart picture on photoprism+ membership page

### DISCUSSION THREAD

**Discussion by somebody-somewhere-over-the-rainbow** [photoprism/photoprism#2375] [General]
Hi all,

could the devs hint me to the source of the [heart image on the membership page](https://dl.photoprism.app/img/artwork/heart-600x600.jpg)?

My child loved the image and I am would like to get a high-res version for a print out ...

Thanks a lot!

**Comment by lastzero** [comment]
Awesome colors, right? Glad we found it in our image collection. Feel free to email sponsors@photoprism.app for details.
